client/src/assets/stylesheets/ui/dropdown.scss

```diff
@@ -0,0 +1,86 @@
+@import './_presets.scss';
+
+.dropdown-wrapper {
+  position: relative;
+  display: inline-block;
+
+  .button {
+    &.tertiary {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+
+  .dropdown-content {
+    position: absolute;
+    padding: 5px 0;
+    border: 1px solid $gray-2;
+    border-radius: $border-radius-default;
+    background-color: #fff;
+    min-width: 150px;
+    max-width: 250px;
+
+    -webkit-box-shadow: 0px 0px 52px -9px rgba(194, 194, 194, 0.8);
+    -moz-box-shadow: 0px 0px 52px -9px rgba(194, 194, 194, 0.8);
+    box-shadow: 0px 0px 52px -9px rgba(194, 194, 194, 0.8);
+
+    &.dropdown-align-left {
+      left: 0;
+    }
+
+    &.dropdown-align-center {
+      left: -15%; // how to center this still needs to be figured out
+    }
+
+    &.dropdown-align-right {
+      right: 0;
+    }
+
+    hr {
+      border-color: $gray-3;
+      margin: 5px 0;
+      border-top: 0;
+      border-style: solid;
+    }
+
+    .dropdown-content-row {
```

Comment:
add background-color: transparent here and it fixes it for me

Reply:
ok I did. check and see if it looks good now.